core/renderer.go

```diff
@@ -0,0 +1,54 @@
+package core
+
+import (
+	"io/ioutil"
+	"strings"
+
+	"github.com/AlecAivazis/survey/terminal"
+	"github.com/chzyer/readline"
+)
+
+// basic readline config that does not put terminal in Raw mode
+// so that basic editing works (backspace, arrow keys etc).  This
+// is used by all text-input prompt types
+var SimpleReadlineConfig = &readline.Config{
```

Comment:
Is there a reason to use this config instead of the one provided by default in `terminal.GetReadline()` which should return a configured instance that is already passed though - i figured you needed a different config here than in the default one but it's not clear to me why